features/step_definitions/web_steps.rb

```diff
@@ -76,7 +76,7 @@ def with_scope(locator)
 
 When /^(?:|I )select "([^\"]*)" from "([^\"]*)"(?: within "([^\"]*)")?$/ do |value, field, selector|
   with_scope(selector) do
-    select(value, :from => field)
+    select(value, :from => field, :disabled => false)
```

Comment:
ummm, i object to this. this step def is used by hundreds of steps in dozens of scenarios; did this change actually pass a regression?  what is the purpose of adding `:disabled => false` here - what problem does that solve?

Reply:
Oops, this was an artifact of trying to debug feature flags. This should have been removed, thank you for catching it!